src/kibana/components/fancy_forms/kbn_model.js

```diff
@@ -24,8 +24,8 @@ define(function (require) {
     var ngModel = this;
 
     // verify that angular works the way we are assuming it does
-    if (String(ngModel.$setViewValue).replace(/\s+/g, '') !== SVV_CHECKSUM) {
-      throw new Error('ngModelController.$setViewValue has updated but KbnModelController has not!');
+    if (angular.version.full !== '1.2.27') {
```

Comment:
Do have 1.2.27 installed locally? the bower.json file mentions `"angular": "~1.2.14"` and that is the version that travis installed. If 1.2.27 is a compatible version we should update the bower file to a locked version number